tests/integration/api_v3/rpc/blockchainApps.test.js

```diff
@@ -42,82 +41,94 @@ describe('get.blockchain.apps', () => {
 		curChainID = response.result.data.chainID;
 	});
 
-	it('returns list of all blockchain applications', async () => {
+	it('should return list of all blockchain applications', async () => {
 		const response = await getBlockchainApps();
 		expect(response).toMap(jsonRpcEnvelopeSchema);
 		const { result } = response;
 		expect(result.data).toBeInstanceOf(Array);
 		expect(result.data.length).toBeGreaterThanOrEqual(1);
 		expect(result.data.length).toBeLessThanOrEqual(10);
-		result.data.forEach(blockchainApp => expect(blockchainApp).toMap(blockchainAppSchema));
-		expect(result.meta).toMap(metaSchema);
+		expect(result).toMap(blockchainAppsSchema);
 	});
 
-	it('returns list of all blockchain applications with limit=10', async () => {
+	it('should return list of all blockchain applications when called with limit=10', async () => {
 		const response = await getBlockchainApps({ limit: 10 });
 		expect(response).toMap(jsonRpcEnvelopeSchema);
 		const { result } = response;
 		expect(result.data).toBeInstanceOf(Array);
 		expect(result.data.length).toBeGreaterThanOrEqual(1);
 		expect(result.data.length).toBeLessThanOrEqual(10);
-		result.data.forEach(blockchainApp => expect(blockchainApp).toMap(blockchainAppSchema));
-		expect(result.meta).toMap(metaSchema);
+		expect(result).toMap(blockchainAppsSchema);
 	});
 
-	it('returns list of all blockchain applications with limit=10 and offset=1', async () => {
+	it('should return list of all blockchain applications when called with limit=10 and offset=1', async () => {
 		const response = await getBlockchainApps({ limit: 10, offset: 1 });
 		expect(response).toMap(jsonRpcEnvelopeSchema);
 		const { result } = response;
 		expect(result.data).toBeInstanceOf(Array);
 		expect(result.data.length).toBeGreaterThanOrEqual(0);
 		expect(result.data.length).toBeLessThanOrEqual(10);
-		result.data.forEach(blockchainApp => expect(blockchainApp).toMap(blockchainAppSchema));
-		expect(result.meta).toMap(metaSchema);
+		expect(result).toMap(blockchainAppsSchema);
 	});
 
-	it('returns list of all blockchain applications by chainID', async () => {
+	it('should return list of all blockchain applications when called by valid chainName', async () => {
+		const response = await getBlockchainApps({ chainName: 'enevti' });
+		expect(response).toMap(jsonRpcEnvelopeSchema);
+		const { result } = response;
+		expect(result.data).toBeInstanceOf(Array);
+		expect(result.data.length).toEqual(1);
+		expect(result).toMap(blockchainAppsSchema);
```

Comment:
We can remove following redundant checks for rpc testcases.
```suggestion
		expect(result).toMap(blockchainAppsSchema);
		expect(result.data.length).toEqual(1);
```